specification/hugr.md

```diff
@@ -1191,7 +1194,18 @@ The new hugr is then derived as follows:
 
 ###### `Replace`
 
-This is the general subgraph-replacement method.
+This is the general subgraph-replacement method. Intuitively, it takes a set of
+sibling nodes to remove and replace with a new set of nodes. The new set of
+nodes is itself a HUGR with some "holes" (edges and nodes that get "filled in"
+by the `Replace` operation). To fully specify the operation, some further data
+are needed:
+
+  - The replacement may containg container nodes with no children, which replace
```

Comment:
`containg` -> `contain` or perhaps `include`. However I'm less sure about these "replace"ing removed container nodes. They do WRT the replaced node's children, but not necessarily in any other way (e.g. they may have different parents, siblings, incoming/outgoing dataflow/controlflow edges to the "replaced" container nodes).

Reply:
How about:
```
 - The replacement may include container nodes with no children, which adopt the children of removed container nodes and prevent those children being removed
```
? - note we haven't said (yet / in the summary) that children of removed nodes are themselves removed